### PR TITLE
WRO-659: Removed `enzyme` and related modules.

### PR DESCRIPTION
**Pull request by hong6316**
### Issue Resolved / Feature Added
Remove enzyme support and dependencies in 5.0.0.

### Resolution
* Removed `enzyme` and related modules

### Links
WRO-659

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

**Comment by MikyungKim** [comment]
Please update your PR's description and resolve the merge conflict.

File: commands/test.js [hunk 1]
```diff
@@ -151,9 +151,9 @@ function assignOverrides(config) {
 function api(args = []) {
 	const config = require('../config/jest/jest.config');
 
-	// Notify users that Enzyme is deprecated and will be removed in Enact CLI 5.0.0.
+	// Notify users that Enzyme was removed in Enact CLI 5.0.0.
 	console.warn(
-		"\nWarning: 'enzyme' support is deprecated and will be removed in Enact CLI 5.0.0. Consider using '@testing-library/react' instead."
+		"\nWarning: 'enzyme' support was removed in Enact CLI 5.0.0. Consider using '@testing-library/react' instead."
```
Comment (MikyungKim) [inline commands/test.js:156]: Please remove the warning entirely.

File: docs/testing-apps.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ order: 7
 The `enact test` command (aliased as `npm run test`) will activate a [Jest](https://jestjs.io/) test runner on all discovered *-specs.js files. All the complicated configuration is hidden away within Enact CLI to avoid any confusion or additional difficulty in testing source code.
 
 Internally Enact CLI supports [React Testing Library](https://testing-library.com/docs/react-testing-library/intro) out of the box, so there's no need to install them locally on your project. Simply import/require it within specs files and it'll be there. You will want to familiarize yourself with React Testing Library's utilities in order to write more complex tests.
-Enzyme support is deprecated and will be removed in a future major release.
+Enzyme support was removed in Enact CLI 5.0.0.
```
Comment (MikyungKim) [inline docs/testing-apps.md:18]: You can just remove the line here.

File: commands/test.js (hunk 1 above)
Comment (hong6316) [reply commands/test.js]: OK

File: docs/testing-apps.md (hunk 2 above)
Comment (hong6316) [reply docs/testing-apps.md]: OK

**Comment by MikyungKim** [comment]
Please update the branch with the recent develolp.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**